.landscaper/container-deployer/blueprint.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+
+imports:
+- name: cluster
+  targetType: landscaper.gardener.cloud/kubernetes-cluster
+- name: releaseName
+  schema:
+    type: string
+- name: releaseNamespace
+  schema: 
+    type: string
+- name: values
+  schema:
+    type: object
+
+deployExecutions:
+- name: default
+  type: GoTemplate
+  template: |
+    deployItems:
+    - name: deploy
+      type: landscaper.gardener.cloud/helm
+      target: 
+        name: {{ .imports.cluster.metadata.name }}
+        namespace: {{ .imports.cluster.metadata.namespace }}
+      config:
+        apiVersion: helm.deployer.landscaper.gardener.cloud/v1alpha1
+        kind: ProviderConfiguration
+        updateStrategy: update
+        name: {{ .imports.releaseName }}
+        namespace: {{ .imports.releaseNamespace }}
+        chart:
+          {{ $resource := getResource .cd "name" "container-deployer-chart" }}
+          ref: {{ $resource.access.imageReference }}
+        values:
+          {{ toYaml .imports.values | indent 4 }}
```

Comment:
the controller image reference `image.repository` should be read from the component descriptor

Reply:
Fixed, image ref will now be taken from component descriptor and merged with the other values for the Helm charts that come from an import.